castervoice/doc/readthedocs/Mouse.md

```diff
@@ -38,34 +38,48 @@ Demonstration [here](https://youtu.be/UISjQBMmQ-I).
+Creates a grid on the screen with horizontal (top of screen) and vertical (left of screen) numbers at the borders. Squares are directly selectable by calling out the respective numbers. 
+
+Selecting from one point to another is available including fine adjustment of the start and end points.
 
 **Usage**:
 
-- Evoke: `douglas`
-- Select square: `<horizontal_number> by <vertical_number>`
+- Evoke: `douglas` or `douglas <monitor_number>`
+- Move to square: `<horizontal_number> by <vertical_number>`
+- Select: `<horizontal_number_1> by <vertical_number_1> select <horizontal_number_2> by <vertical_number_2>`
+- Select horizontally: `<horizontal_number_1> by <vertical_number_1> select <horizontal_number_2>`
+- Fine selection: `<horizontal_number_1> by <vertical_number_1> move` &rightarrow; `curse ...` &rightarrow; `point one` &rightarrow; `<horizontal_number_2> by <vertical_number_2> move` &rightarrow; `curse ...` &rightarrow; `point two` &rightarrow; `select`
 
-**Example**:
+**Examples**:
 
 - _5 by 20_
+- _5 by 20 kick_
+- _5 by 20 select 10 by 30_
+- _5 by 20 select 10_
+- _5 by 20 move, curse sauce lease 5, point one, 10 by 30 move, curse dunce lease 10, point two, select_
 
 ### Rainbow
 
 **Description**:
 
-- Creates a colored grid on the screen. Squares are directly selectable by calling out the respective color and number.
+- Creates a colored grid on the screen. Squares are directly selectable by calling out the respective color then number.
 
 **Usage**:
 
-- Evoke: `rainbow`
-- Select square: `[<number_of_color_palette] <color> <number>`  
+- Evoke: `rainbow` or `rainbow <monitor_number>` 
+- Move to square: `[<number_of_color_palette>] <color> <number>`  
   _number_of_color_palette_ refers to the fact that colors loop when filling the screen.  
   _color_ can be replaced by the following colors: _red_, _orange_, _yellow_, _green_, _blue_, _purple_.
+- Select: `[<number_of_color_palette_1>] <color_1> <number_1>` select `[<number_of_color_palette_2>] <color_2> <number_2>`
+- Fine selection: `[<number_of_color_palette_1>] <color_1> <number_1> move` &rightarrow; `curse ...` &rightarrow; `point one` &rightarrow; `[<number_of_color_palette_2>] <color_2> <number_2> move` &rightarrow; `curse ...` &rightarrow; `point two` &rightarrow; `select`
 
-**Example**:
+**Examples**:
 
-- _red 86_: - References the according square within the first color palette.
-- _two red 86_ - References the according square within the second color palette.
+- _red 86_: - References square within the first red palette.
+- _red 86 kick_: - Clicks at this location.
+- _two red 86_ - References square within the second red palette.
+- _red 86 select 2 green 10_: - Selects from the first red 86 to the second green 10.
+- _red 86, curse sauce lease 5, point one, select 2 green 10, curse dunce lease 10, point two, select_: - Selects from the first red 86 to the second green 10 with fine adjustment.
```

Comment:
Should it be, 
"_red 86 move, curse sauce lease 5, point one, 2 green 10 move, curse dunce lease 10, point two, select_"